domain_scoring/domain_scoring.py

```diff
@@ -112,3 +114,44 @@ def _extract_data_labels(self, metapath_graph: MetaPathRatingGraph) -> (List[Tup
             metapath_labels.append(LARGER)  # >
 
         return metapath_pairs, metapath_labels
+
+    def _test_score(self, x_test, y_test):
+        print('Test accuracy is {}'.format(self.classifier.score(X=self._preprocess(x_test), y=y_test)))
```

Comment:
This doesn't always yield the accuracy as we say with the RandomForestRegressor

Reply:
You mean that this depends on the predictor?

Reply:
Yes, either we we don't call `self.classifier.score` and instead predict the labels with our classifier and evaluate it with the general scikit-learn `accuracy()` method or we print "Test score is..." but that's more or less useless if you don't know which score this is.